### ISSUE TITLE
Enable support for torch.maximum

### ISSUE THREAD

**Issue by bringlein** [torch-spyre/torch-spyre#2187]
### What feature would you like to be added?

[`torch.maximum`](https://docs.pytorch.org/docs/2.12/generated/torch.maximum.html)

and please note, `torch.maximum != torch.max` (https://github.com/torch-spyre/torch-spyre/issues/453). 

### Why is this feature needed?

For online softmax (e.g. used in flash attention or paged attention) we would need the element-wise `torch.maximum`, see e.g. https://github.com/torch-spyre/spyre-inference/pull/201

**Comment by avery-blanchard** [comment]
Support added in #2153

**Comment by bringlein** [comment]
thanks, our first tests confirms it works (I searched for `torch.maximum` in the issues...there the PR didn't show up :D )
